cli/azd/pkg/apphost/generate.go

```diff
@@ -778,6 +780,9 @@ func injectValueForBicepParameter(resourceName, p string, parameter any) (string
 	if p == knownParameterLogAnalytics {
 		return knownInjectedValueLogAnalytics, true, nil
 	}
+	if p == knownParameterContainerEnvName {
```

Comment:
Lets add the containerAppEnvironmentId as well

Reply:
added